sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -116,7 +118,13 @@ object AkkaserverlessPlugin extends AutoPlugin {
     Test / managedSources :=
       (Test / managedSources).value.filter(s => !isIn(s, (Test / temporaryUnmanagedDirectory).value)),
     Test / unmanagedSources :=
-      (Test / generateUnmanaged).value ++ (Test / unmanagedSources).value)
+      (Test / generateUnmanaged).value ++ (Test / unmanagedSources).value,
+    onlyUnitTest := JBoolean.getBoolean("onlyUnitTest"),
```

Comment:
this doesn't work with how it's documented

`-DonlyUnitTest` will add the system property but the value will be `""`

it would have to be `-DonlyUnitTest=true`, but maybe better to change this again so that both `""` and `true` is supported

Reply:
Ups. 
OK, I'll make `-DonlyUnitTest` behave as `-DonlyUnitTest=true` and update the docs accordingly.